docs/gateway/b2.md

```diff
@@ -43,6 +43,7 @@ mc ls myb2
 ### Known limitations
 Gateway inherits the following B2 limitations:
 
+- PutObject() does not return an md5sum of the uploaded file as an etag response.  Apps that check vailidity will fail.
```

Comment:
Modified your documentation a little bit. PTAL.
```
PutObject S3 API does not return md5sum of the uploaded file as it's etag response. Applications that check validity of the uploaded object by comparing these two values will fail.
```

Reply:
Looks good!

Reply:
Can you update the PR with the requested change?